docs/architecture/main.md

```diff
@@ -0,0 +1,1259 @@
+# **About arc42**
```

Comment:
regarding the images under architecture/images: with reference to this TRG https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4, would it be possible to move of the diagrams to mermaid or similar or to provide also some file which would allow editing in the future? as PNGs don't allow editing

Reply:
There is one image system_scope.png that needs to be converted in diagram as code. we will check if this can be converted using mermaid. Is it okay if we push this change in the separate PR?

For add sequence diagram image plant UML is added
Other screenshots are added from swagger documentation.